### PR TITLE
Update jest snapshots & cypress tests

### PR DESCRIPTION
**Pull request by ps48**
### Description
Update jest snapshots & cypress tests

### Issues Resolved
* update data ingestion
* update date time changes
* move jaeger data file to fixtures  
* Update jest snapshots due to upstream changes
* Remove all explicit wait and delay calls

### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass, including unit test, integration test and doctest
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
  - [ ] New functionality has user manual doc added
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: gantt-chart/.cypress/integration/ui.spec.js [hunk 1]
```diff
@@ -5,35 +5,31 @@
...
-import {
-  testDataSet,
-  delay,
-  GANTT_VIS_NAME,
-  Y_LABEL,
-  X_LABEL,
-  DEFAULT_SIZE,
-} from '../utils/constants';
+import customParseFormat from 'dayjs/plugin/customParseFormat';
+import dayjs from 'dayjs';
```
Comment (joshuali925) [inline gantt-chart/.cypress/integration/ui.spec.js:9]: are we moving to dayjs or just for testing?

**Review by joshuali925 — APPROVED** [review]

File: gantt-chart/.cypress/integration/ui.spec.js [hunk 2]
```diff
@@ -51,39 +47,36 @@ describe('Dump test data', () => {
...
     cy.get('.euiButton__text').contains('Create ').click({ force: true });
-    cy.wait(delay * 3);
-    cy.get('span[data-test-subj="visTypeTitle"]').contains('Gantt Chart').click({ force: true });
-    cy.wait(delay * 3);
-    cy.get('.euiListGroupItem__label')
-      .contains(/^jaeger$/)
-      .click({ force: true });
-    cy.wait(delay * 5);
-    cy.get('.euiButton__text').contains('Save').click({ force: true });
+    cy.wait(delay);
```
Comment (mengweieric) [inline gantt-chart/.cypress/integration/ui.spec.js:54]: We may want to remove explicit wait

**Review by mengweieric — APPROVED** [review]

**Review by YANG-DB — APPROVED** [review]

File: gantt-chart/.cypress/integration/ui.spec.js (hunk 1 above)
Comment (ps48) [reply gantt-chart/.cypress/integration/ui.spec.js:9]: This is just for testing time formats. We had issues earlier when checking absolute time, hence moved to testing time format. FTR has this issue already fixed in.

File: gantt-chart/.cypress/integration/ui.spec.js (hunk 2 above)
Comment (ps48) [reply gantt-chart/.cypress/integration/ui.spec.js]: removed all explicit waits

**Review by mengweieric — APPROVED** [review]

**Review by joshuali925 — APPROVED** [review]

**Pull request merged**